be/src/storage/persistent_index.cpp

```diff
@@ -531,57 +593,62 @@ Status ImmutableIndexWriter::write_shard(size_t key_size, size_t npage_hint, siz
     return Status::OK();
 }
 
-Status ImmutableIndexWriter::write_shard_as_rawbuff(const ImmutableIndex::ShardInfo& old_shard_info,
-                                                    ImmutableIndex* immutable_index) {
-    const auto key_size = old_shard_info.key_size;
-    const bool new_key_length = _nshard == 0 || _cur_key_size != key_size;
-    if (_nshard == 0) {
-        _cur_key_size = key_size;
-        _cur_value_size = kIndexValueSize;
-    } else {
-        if (new_key_length) {
-            CHECK(key_size > _cur_key_size) << "key size is smaller than before";
+Status ImmutableIndexWriter::write_bf() {
+    size_t pos_before = _idx_wb->size();
+    LOG(INFO) << "write kv size:" << pos_before << ", _bf_wb size: " << _bf_wb->size();
+    if (_bf_wb->size() != 0) {
+        LOG(INFO) << "_bf_wb already write size: " << _bf_wb->size();
```

Comment:
This may add to many logs, if this is necessary, get those info and put into caller's INFO log.